phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -79,13 +81,28 @@ public void showAvailableApps() {
      */
     public void populate(List<CategoryDTO> categories) {
         Platform.runLater(() -> {
+            Set<String> themedCategories = Sets.newHashSet(
+                    "custom",
```

Comment:
I think that any categories can not be themed by a given theme. This is important not to make particular cases in the code